jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -108,6 +115,13 @@ def _sync_available_from_state(self, *args):
                              self.subset_select._subset_to_dict(subset) for subset in
                              self.data_collection.subset_groups]
 
+    @observe('multiselect')
+    def _on_multiselect_change(self, change=None):
+        if not self.multiselect and isinstance(self.subset_selected, List):
+            self.subset_selected = self.subset_selected[0]
```

Comment:
I think this should be handled by `SelectPluginComponent._multiselect_changed` already.

Reply:
Alright, I'll try taking it out and seeing if it still works. For some reason I think I had to set that manually before but maybe that was user error.